scss/_button-group.scss

```diff
@@ -72,8 +72,8 @@
 //
 
 .dropdown-toggle-split {
-  padding-right: $btn-padding-x * .75;
-  padding-left: $btn-padding-x * .75;
+  padding-right: $dropdown-toggle-split-padding-right;
```

Comment:
These values should likely always be the same. Let's stick with one `-x` in the variable name instead of two variable with `-right` and `-left`.